### PR TITLE
server: tikv reports usedSize.

### LINKED ISSUE #587: pd metrics pd_cluster_status{type="storage_size"} is incorrect
If tikv is not deployed in a standalone disk, `pd_cluster_status{type="storage_size"}` metrics would be incorrect. For example, a newly deployed cluster shows `Current Storage Size` 243.7GB on grafana, this is obviously nonsense

### PR DESCRIPTION
**Pull request by disksing**
Fix #587
/cc @andelf @nolouch @zhangjinpeng1987

### CODE REVIEW

**Comment by andelf** [comment]
LGTM

File: server/filter.go [hunk 1]
```diff
@@ -159,7 +159,7 @@ func (f *snapshotCountFilter) FilterTarget(store *storeInfo) bool {
 // storageThresholdFilter ensures that we will not use an almost full store as a target.
 type storageThresholdFilter struct{}
 
-const storageRatioThreshold = 0.8
+const storageRatioThreshold = 0.2
```
Comment (siddontang) [inline server/filter.go]: why change to 0.2?

Comment (disksing) [inline server/filter.go]: It's for checking if ok to move data to the store. It should depend on available space.

File: server/filter.go [hunk 2]
```diff
@@ -170,7 +170,7 @@ func (f *storageThresholdFilter) FilterSource(store *storeInfo) bool {
 }
 
 func (f *storageThresholdFilter) FilterTarget(store *storeInfo) bool {
-	return store.storageRatio() > storageRatioThreshold
+	return store.availableRatio() < storageRatioThreshold
```
Comment (siddontang) [inline server/filter.go]: no need new test?

Comment (disksing) [inline server/filter.go]: Looks I should add a test case that usedSize+available!=capacity

**Comment by siddontang** [comment]
Please fix the conflict.

**Comment by disksing** [comment]
PTAL @siddontang

File: server/filter.go (hunk 1 above)
Comment (siddontang) [inline server/filter.go]: storageRatio looks like current storage size / storage capacity, but here is for available storage size / storage capacity, maybe we should rename to storageAvailableRatio?

Comment (disksing) [inline server/filter.go]: good idea.

**Pull request merged**